dataaccess/mongo_db.py

```diff
@@ -110,6 +110,17 @@ def get_number_of_nubela_api_requests_last_hour(self, email: str) -> int:
         }
 
         return coll.count_documents(query)
+    
+    def get_previous_phishing_email_generated_by_user(self, email: str) -> list[dict]:
+        collection = 'phishes'
+        coll = self._db[collection]
+
+        query = {
+            'requester.email': email
+        }
+        projection = {'mail': 1, 'linkedin_data.public_identifier': 1, 'profile_image': 1, 'subject':1, '_id': 1}
```

Comment:
Not really a problem but _id automatically is 1 even if you don't write it. Normally you say '_id': 0 if you don't want it